app/Actions/Teams/JoinTeamAction.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Actions\Teams;
+
+use App\Models\Teams\Team;
+use App\Models\User\User;
+
+class JoinTeamAction
+{
+    public function run(User $user, Team $team)
```

Comment:
Actually, all other big OSS projects refrain from using redundant comment blocks. The argument is that they make the code less readable and potentially confuse the readers, especially when the class/method action is self-explanatory. This is coming from the legends in the Laravel industry https://spatie.be/guidelines/laravel-php#docblocks 
I can add comments for everything from now on, but it would only make the codebase worse, in my opinion (and probably the community's opinion).
Also, let's add a PHP style-ci by the upcoming releases, and unify the whole project's style guide. We'll use https://github.com/FriendsOfPHP/PHP-CS-Fixer